app/views/search/_delay_modal.html.erb

```diff
@@ -0,0 +1,24 @@
+<!--begin modal-->
+<div class="modal" id="delayWarningModal" tabindex="-1" role="dialog" aria-labelledby="delayWarningModal">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <a href="#" class="close modal-close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></a>
+        <h4 class="modal-title" id="delayWarningModalLabel">Possible Delay Ahead</h4>
+      </div>
+      <div class="modal-body">
+        <p>
+          This collection is known to cause delays and may take longer to download than expected.
+        </p>
+        <p>
```

Comment:
Obviously I've not seen this in context of the app/workflow... but I think we should style this optional message from the provider a bit differently or give the user some context that the message comes from the provider. Maybe a "Message from data provider" label/heading?

Reply:
@mreese84 I took a shot at it - added some leading text to make it more obvious where the optional message was coming from.

Let me know what you think ;)